app/remote/hooks/rapidpro/RapidProWebhook.ts

```diff
@@ -30,6 +33,42 @@ export default class RapidProWebhook implements IWebhookRepository {
         await this.http.post(this.baseCallbackUrl + '/closeRoom', reqOptions);
```

Comment:
You don't need append anything to `baseCallbackUrl` because there is already a `type` field in payload..